secret_santa/api/models.py

```diff
@@ -1,3 +1,79 @@
+import random
+import uuid
+
 from django.db import models
 
 # Create your models here.
+
+
+STATUS_CHOICES = (
+    ("open", "Open"),
+    ("moving", "Moving"),
+    ("close", "Close"),
+)
```

Comment:
Can we use enum?

Reply:
Didn't understand what you mean

Reply:
Okay, i find what you mean, i'll try this

Reply:
As i read about Enum, does not correctly support the HTML support render of the REST Framework

Reply:
Can you create enum with class method `choices` which return structure like you describe above?